.sonarlint/nanoframework_system.device.pwmcsharp.ruleset

```diff
@@ -0,0 +1,365 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
neither this one :-)

Reply:
Yes, those are the sonarcube rules file, allowing to run the analysis locally. If the dev doesn't have their extension installed it's OK. They are are harmless.

Reply:
More details [here](https://stackoverflow.com/questions/37901733/sonar-lint-ruleset-files-gitignore)